UPGRADE-4.0.md

```diff
@@ -288,9 +288,9 @@ Validator
        // ...
    }
    ```
-   
+
  * The default value of the strict option of the `Choice` Constraint has been
-   changed to `true` as of 4.0. If you need the the previous behaviour ensure to 
+   changed to `true` as of 4.0. If you need the the previous behaviour ensure to
```

Comment:
Good catch, but not related ;) see https://github.com/symfony/symfony/pull/21250